async-download-rest/src/main/java/org/uniprot/api/async/download/messaging/listener/idmapping/IdMappingMessageListener.java

```diff
@@ -41,25 +43,25 @@ public class IdMappingMessageListener extends BaseAbstractMessageListener
     private final IdMappingDownloadResultWriterFactory writerFactory;
 
     public IdMappingMessageListener(
-            DownloadConfigProperties downloadConfigProperties,
-            AsyncDownloadQueueConfigProperties asyncDownloadQueueConfigProperties,
+            DownloadConfigProperties idMappingDownloadConfigProperties,
```

Comment:
For DownloadConfigProperties and HeartBeatProducer, we can use the concrete types here to mitigate any conflict in the future. Check for UniProtKB and uniref too